client/dashboard/sites/logs/dataviews/style.scss

```diff
@@ -47,3 +47,22 @@
 		color: #2c3338;
 	}
 }
+
+/* Scroll to top button */
+.site-logs-card .site-logs-scroll-to-top {
+	position: fixed;
+	inset-block-end: 2rem;
+	inset-inline-end: 2rem;
+	background: var( --color-primary );
+	color: var( --studio-white );
```

Comment:
We should not use these color variables. They come from the color schemes and are not introduced in HD v2.